SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalanceEnforcer/SpatialLoadBalanceEnforcerTest.cpp

```diff
@@ -45,71 +45,28 @@ void AddEntityToStaticComponentView(USpatialStaticComponentView& StaticComponent
 	}
 }
 
-void InitialiseVirtualWorkerTranslator(SpatialVirtualWorkerTranslator* VirtualWorkerTranslator, bool bAddDefaultWorkerMapping)
+void InitialiseVirtualWorkerTranslator(SpatialVirtualWorkerTranslator* VirtualWorkerTranslator)
 {
 	Schema_Object* DataObject = TestingSchemaHelpers::CreateTranslationComponentDataFields();
-	if (bAddDefaultWorkerMapping)
-	{
-		TestingSchemaHelpers::AddTranslationComponentDataMapping(DataObject, 1, ValidWorkerOne);
-		TestingSchemaHelpers::AddTranslationComponentDataMapping(DataObject, 2, ValidWorkerTwo);
-	}
+
+	TestingSchemaHelpers::AddTranslationComponentDataMapping(DataObject, 1, ValidWorkerOne);
```

Comment:
This is my bad but we could change 1 and 2 here for VirtualWorkerOne  and VirtualWorkerTwo <3

Reply:
done